meta-intel-extras/conf-qt/conf-notes.txt

```diff
@@ -1,5 +1,5 @@
 Available PELUX images:
-* core-image-pelux
-* core-image-pelux-qtauto
+* core-image-pelux-minimal
+* core-image-pelux-qtauto-neptune
```

Comment:
Is there a reason you have added neptune to the end here?

Reply:
Yes. With the bbclass-ification of the qtauto and pelux images, the image names have changed.

Reply:
Maybe the name of the bbclass should change instead? Adding neptune is redundant and confusing and the image names will be seen by more people than the names of the bbclasses.

Reply:
Thanks Tobias. 

Come to think about it: Qt Auto is a framework and Neptune is just the reference UI so it would kinda make sense to have it explicitly in the image name. You could have a Qt Auto Pelux with your own SysUI and App implementations.